site/en/blog/migrating-to-typescript/index.md

```diff
@@ -11,12 +11,15 @@ alt: ''
 tags:
   - devtools-engineering
   - devtools
+  - devtools-architecture
 ---
 
 {% include 'partials/devtools/en/banner.md' %}
 
 <!-- lint disable no-smart-quotes -->
```

Comment:
unrelated nit: you should be able to remove this now, we're changing the checker only to run in JS blocks

Reply:
Nice. I'll send another PR for this. Quite a few DevTools posts disable that.